.gitignore

```diff
@@ -13,17 +14,8 @@ cabal.project.local
 # Stack
 .stack-work
 
-# Emacs
-TAGS
-.dir-locals.el
-
-# Vim
-tags
-*.swp
-
-# Intellij
-*.iml
-.idea/
```

Comment:
I don't understand why you removed these entries from `.gitignore`. Can you elaborate?

Reply:
Because these should go into user `.gitignore` and are not project-specific. cc @kirelagin